howto/hybrid-clusters/README.md

```diff
@@ -0,0 +1,28 @@
+# Deploying to Hybrid Linux/Winodws K8s Clusters
+
+## Installing the Dapr Control Plane
+
+If you are installing using the Dapr CLI or via helm chart, you can simply follow our normal deployment procedures:
+[Installing Dapr on a Kubernetes cluster](../../getting-started/environment-setup.md#installing-Dapr-on-a-kubernetes-cluster)
+
+Affinity will be automatically set for kubernetes.io/os=linux. If you need to override linux to another value, you can do so by setting:
+```
+helm install dapr dapr/dapr --set global.daprControlPlaneOs=YOUR_OS
+```
+Dapr control plane container images are only provided for Linux, so you shouldn't need to do this unless you really know what you are doing.
+
+## Installing Dapr Apps
+The Dapr sidecar container is currently Linux only. For this reason, if you are writing a Dapr application, you must run it in a Linux container. 
+> **Note:** Windows support for dapr applications is in progress. Please see: [dapr/dapr#842](https://github.com/dapr/dapr/issues/842).
+
+When deploying to a hybrid cluster, you must configure your apps to be deployed to only Linux available nodes. One of the simplest ways to do this is to add kubernetes.io/os=linux to your app's nodeSelector.
+
+```yaml
+spec:
+  nodeSelector:
+    kubernetes.io/os: linux
+```
+
+Kubernetes also supports much more advanced configuration via node affinity. 
+See https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/ for more examples.
+
```

Comment:
What's the next logical step for someone to go after reading this? 
If there are related links please add "Related Links" section as described in the guidance: https://github.com/dapr/docs/tree/master/contributing#contributing-to-how-tos

Reply:
I am aware of that template. In 0.9, there just isn't anywhere to go. This document is currently just a stub for a feature we do not yet support. I'll be adding a full howto in conjunction with the referenced epic.